cli/board/attach.go

```diff
@@ -16,69 +16,60 @@
 package board
 
 import (
-	"context"
 	"fmt"
 	"os"
 
 	"github.com/arduino/arduino-cli/cli/arguments"
 	"github.com/arduino/arduino-cli/cli/errorcodes"
 	"github.com/arduino/arduino-cli/cli/feedback"
-	"github.com/arduino/arduino-cli/cli/instance"
-	"github.com/arduino/arduino-cli/cli/output"
-	"github.com/arduino/arduino-cli/commands/board"
-	rpc "github.com/arduino/arduino-cli/rpc/cc/arduino/cli/commands/v1"
-	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-var (
-	port arguments.Port
-)
-
 func initAttachCommand() *cobra.Command {
+	var port arguments.Port
 	attachCommand := &cobra.Command{
-		Use:   fmt.Sprintf("attach -p <%s>|-b <%s> [%s]", tr("port"), tr("FQBN"), tr("sketchPath")),
+		Use:   fmt.Sprintf("attach [-p <%s>] [-b <%s>] [%s]", tr("port"), tr("FQBN"), tr("sketchPath")),
```

Comment:
```suggestion
		Use:   fmt.Sprintf("attach [-p <%s> [-l <%s>]] [-b <%s>] [%s]", tr("port"), tr("protocol"), tr("FQBN"), tr("sketchPath")),
```

It seems to me this effectively communicates the nature of the `-l` flag as a supplement to the `-p` flag, which may only be used in combination with that flag.

Reply:
Nice!